server/src/controllers/user.js

```diff
@@ -15,7 +19,7 @@ const createUser = async (req, res, next) => {
 const updateUser = async (req, res) => {
   try {
     const user = await User.findById(req.params.id);
-  } catch (error) {}
+  } catch (error) { }
```

Comment:
There's no change in this line. Why did you add a space? Can you instead use a formatter and fix this?